README.md

```diff
@@ -7,125 +7,138 @@ up a pre-defined Inventory.  For each of the hosts in such an inventory, Ansible
 will:
 
 - create a Multipass VM with the name given in the inventory
-- add to the VM a user account named "ansible" with SSH access Public Key
-- learn the VM IP address and store it in the in-memory inventory so that
+- add an "ansible" user that has SSH access to the VM
+- learn the VMs IP address and store it in the in-memory inventory so that
   Ansible can run tasks on it
 
-You can see how this is achieved by reviewing the make-multipass-inventory.yaml
-play and templates/cloud-init.yaml.j2
+You can see how this is achieved by reviewing the below-listed files:
 
+- [make-multipass-inventory.yaml](make-multipass-inventory.yaml)
+- [templates/cloud-init.yaml.j2](templates/cloud-init.yaml.j2)
+
+## Prerequisites
+
+- Ubuntu host
```

Comment:
Any Linux host should work here, as snap is installable across most distros (if not all).

Reply:
Got it Chief! I've removed Ubuntu from the list of prerequisites.